cortex-m-demo/src/application.rs

```diff
@@ -312,9 +264,13 @@ where
             None => return Err(Error::InvalidState),
         }
 
-        let my_wire_address = self.config.participants[0].into();
+        const PEER0: [u8; 20] = [
```

Comment:
It might be a good idea to still read this from `self.config`, as currently this value is hard-coded in two places (or put them in a global constant).

Reply:
I agree. Done.